tests/DateSpanTest.php

```diff
@@ -29,8 +29,11 @@
  * @package Date
  * @author Leandro Lucarella <llucax@php.net>
  */
-class Date_SpanTest extends PHPUnit_Framework_TestCase {
-
+class Date_SpanTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Date_Span
+     */
     var $time;
 
     function setUp() {
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.